internal/handlers/common/aggregations/stages/collstats.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package aggregations
+package stages
```

Comment:
`collStatsStage` in this file is still called so. Seeing at how other stages are called, this one should be just `collStats`, right?

Reply:
renamed `collStatsStage` to `collStats`